modules/DiffusionSolver/Solver/general.cc

```diff
@@ -126,17 +127,14 @@ void chi_diffusion::Solver::GetMaterialProperties(const chi_mesh::Cell& cell,
     bool transportxs_found = false;
     for (int p=0; p<material->properties_.size(); p++)
     {
-      if (std::dynamic_pointer_cast<chi_physics::TransportCrossSections>
+      if (std::dynamic_pointer_cast<chi_physics::MultiGroupXS>
```

Comment:
There is no need for the diffusion solver to deal with `MultiGroupXS` is there? We could change it to the base.